tests/baselines/reference/issue52182.js

```diff
@@ -0,0 +1,22 @@
+//// [tests/cases/compiler/issue52182.ts] ////

```

Comment:
This test name is a placeholder I forgot to change; will modify.

(Honestly I wish we just used `issue#` file names so I didn't have to dream up names, but, there aren't any like that in the repo.)